srcpkgs/LACT/files/lact/run

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+exec 2>&1
+exec chpst -n -10 lact daemon
```

Comment:
use a nice value of -10 because that is what's used in the sample systemd service